Bigtable/src/DataClient.php

```diff
@@ -249,14 +250,17 @@ public function readRows(array $options = [])
     {
         $rowKeys = $this->pluck('rowKeys', $options, false) ?: [];
         $ranges = $this->pluck('rowRanges', $options, false) ?: [];
+        $filter = $this->pluck('filter', $options, false) ?: null;
 
         array_walk($ranges, function (&$range) {
             $range = $this->serializer->decodeMessage(
                 new RowRange(),
                 $range
             );
         });
-
+        if (is_string($rowKeys)) {
```

Comment:
I am curious about this. Do we need this as we have `readRow` below which will accept a single row key as a string?

Reply:
I think its better to have this or we should throw exception if rowkeys is not array. What do you prefer? Since this is part of options user can accidentally pass just single row key, In that case it doesnt give proper error. It gives protobuf serialization exception.

Reply:
Good point, I lean towards throwing an `InvalidArgumentException`.

Reply:
updated